### PR TITLE
Egar 598 arrival/departure details pages

### PR DESCRIPTION
**Pull request by nigelhughes552**
**What changes does this PR bring?**

**Definition of Done**
The following need to be checked off as they are complete as part of the
PR process. If you are unable to mark off an item as complete, then state
in the comments why this is the case.
- [x] Have you built the code locally and confirmed its working?
- [ ] Is the build confirmed to be passing on CI?
- [ ] Have you added enough relevant unit tests for your change?
- [ ] Have you added enough relevant E2E tests for your change?
- [ ] Have you updated any relevant documentation as part of this change?
- [ ] Has this change been tested against the Acceptance Criteria?
- [ ] Have you considered how this will be deployed in SIT/Staging/Production?

### CODE REVIEW

**Comment by nigelhughes552** [comment]
Note: As the Departure and Arrival pages are 'stand alone' pages it is possible to enter an Arrival date that is before the Departure date set on the Departure page. Guess the Departure page somehow needs to pass date entered so that when the Arrival page is loaded it can refer to the entered departure date and act accordingly. Not sure how this would be done!

I also noted that before I added the validation to the lat/long fields that the data does not appear to be html encoded - I was able to enter <script> and <script> reappeared after post back. Not sure how much of a hacker security risk this is, I'm no expert, but I was of the understanding that all user input should be sanitised. Maybe this is something that is planned to be done at later stages - just wanted to share my findings.

File: src/app/garfile/arrival/post.controller.js [hunk 1]
```diff
@@ -19,38 +19,81 @@ module.exports = (req, res) => {
...
 
   // Define port / date validation msgs
-  const portMsg = "If the Arrival port is 'ZZZZ' then you must also provide decimal coordinates to 4 decimal places";
+  const portMsg = 'If you do not have the airport code, enter \'ZZZZ\' and enter the latitude and longitude to 4 decimal places below.';
+  const portCodeMsg = 'The arrival airport code must be a minimum of 3 letters and a maximum of 4 letters.';
   const futureDateMsg = 'Arrival date must be today or in the future';
   const realDateMsg = 'Enter a real Arrival date';
   const timeMsg = 'Enter a real Arrival time';
+  const latitudeMsg = 'Value entered is incorrect. Please enter latitude to 4 decimal places.';
+  const longitudeMsg = 'Value entered is incorrect. Please enter longitude to 4 decimal places.';
```
Comment (Meepit) [inline src/app/garfile/arrival/post.controller.js]: Remove the word please. GDS doesn't recommend you use it

File: src/app/garfile/departure/post.controller.js [hunk 2]
```diff
@@ -19,38 +19,81 @@ module.exports = (req, res) => {
...
-  const validations = validator.genValidations(validator.notEmpty, validationIds, validationValues, validationMsgs);
 
   // Define port / date validation msgs
-  const portMsg = "If the Departure port is 'ZZZZ' then you must also provide decimal coordinates to 4 decimal places";
+  const portMsg = 'If you do not have the airport code, enter \'ZZZZ\' and enter the latitude and longitude to 4 decimal places below.';
+  const portCodeMsg = 'The departure airport code must be a minimum of 3 letters and a maximum of 4 letters.';
   const futureDateMsg = 'Departure date must be today or in the future';
   const realDateMsg = 'Enter a real Departure date';
   const timeMsg = 'Enter a real Departure time';
+  const latitudeMsg = 'Value entered is incorrect. Please enter latitude to 4 decimal places.';
```
Comment (Meepit) [inline src/app/garfile/departure/post.controller.js]: Remove the word please

**Comment by Meepit** [comment]
Can you please write a couple test cases for your lattitude / longitude validators?
Here are some example tests for other validators https://github.com/UKHomeOffice/egar-public-site-ui/blob/master/src/test/validators.test.js#L137

File: src/app/garfile/arrival/post.controller.js (hunk 1 above)
Comment (nigelhughes552) [reply src/app/garfile/arrival/post.controller.js]: Should be done

File: src/app/garfile/departure/post.controller.js (hunk 2 above)
Comment (nigelhughes552) [reply src/app/garfile/departure/post.controller.js]: Should be done

**Pull request merged**